daffodil-runtime1-unparser/src/main/scala/org/apache/daffodil/unparsers/runtime1/BinaryNumberUnparsers.scala

```diff
@@ -122,7 +122,7 @@ class BinaryIntegerRuntimeLengthUnparser(
 ) extends BinaryIntegerBaseUnparser(e)
   with HasRuntimeExplicitLength {
 
-  override val runtimeDependencies = Vector(lengthEv)
+  override def runtimeDependencies = Vector(lengthEv)
```

Comment:
There's still a number of `runtimeDependencies` that are vals:
```
$ grep -R 'val runtimeDependencies' | wc -l
88
```

Should those be changed to a def too?

Reply:
Sorry I missed these. Will do!